extensions/typescript-language-features/src/features/directiveCommentCompletions.ts

```diff
@@ -30,6 +30,11 @@ const directives: Directive[] = [
 		description: localize(
 			'ts-ignore',
 			"Suppresses @ts-check errors on the next line of a file.")
+	}, {
```

Comment:
Also, should this be gated to TS 3.9+

Reply:
Here's an example of how we implement version gating: https://github.com/microsoft/vscode/blob/459939b9051e86653ef49f910c408a2b64f27bff/extensions/typescript-language-features/src/features/definitions.ts#L24

Reply:
Thanks! That saved me some time peeking through 😄.

I don't know if this was an intention decision before, but a bug note: the directives don't yet check version even though [`// @ts-ignore` was added in TS 2.6](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html#suppress-errors-in-ts-files-using--ts-ignore-comments) and [`// @ts-check` was added in TS 2.5](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-5.html#type-assertioncast-syntax-in-checkjsts-check-mode). I started to account for that (42ab58246516b5c20e89a6767eefe526e22bfa51) but figured the code was getting a little messy... and went back to keeping the bug (05ed591cf21079f86a392731f4ee7078297cc7f8).